python/py-spopt/Portfile

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        pysal spopt 0.5.0 v
+revision            0
+name                py-${github.project}
+github.tarball_from tarball
+
+categories-append   gis
+
+platforms           {darwin any}
+supported_archs     noarch
+license             BSD
+maintainers         nomaintainer
+
+description         Spatial Optimization (spopt)
+long_description    {*}${description} is an open-source Python library for solving optimization \
+                    problems with spatial data. Originating from the region module in PySAL \
+                    (Python Spatial Analysis Library), it is under active development for the \
+                    inclusion of newly proposed models and methods for regionalization, facility \
+                    location, and transportation-oriented solutions.
+
+homepage            https://github.com/pysal/spopt
+
+checksums           rmd160  4d387eb900763e5e49b41b1f68ace0c0aa3a029e \
+                    sha256  c3d6572c1167b5d59a98781f8697cc6ba10f1fbf65ecf1ca1bad1aaa9d4ce511 \
+                    size    29182492
+
+python.versions     39 310 311
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools \
+                    port:py${python.version}-setuptools_scm
+
+    depends_test-append \
+                    port:py${python.version}-pytest \
```

Comment:
this is added by default when setting `test.run yes` so the `pytest` dependency can be removed here